ifrs17/Constants/Validations.ipynb

```diff
@@ -88,11 +88,7 @@
                 "\n    };",
                 "\n    ",
                 "\n}"
-            ],
-            "metadata": {},
-            "execution_count": 0,
-            "outputs": []
-        },
+
```

Comment:
also this extra line. It's better to remove it...

Reply:
gone now after the correct merger has been properly redone.